src/Sylius/Bundle/CoreBundle/Migrations/Version20220210135918.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220210135918 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add by_guest field to mark orders made by guests';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE sylius_order ADD by_guest TINYINT(1) DEFAULT \'1\' NOT NULL');
```

Comment:
```suggestion
        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');

        $this->addSql('ALTER TABLE sylius_order ADD by_guest TINYINT(1) DEFAULT \'1\' NOT NULL');
```

Reply:
this field is not-nullable, what will happen with already placed orders or carts? should we prepare migration for those orders?

Reply:
Added 👌🏻